packages/neovim/src/attach/attach.test.ts

```diff
@@ -155,12 +155,15 @@ describe('Nvim API', () => {
   });
 
   it('emits "disconnect" after quit', done => {
-    const disconnectMock = jest.fn();
-    nvim.on('disconnect', disconnectMock);
+    let called = false;
+    nvim.on('disconnect', () => {
+      called = true;
+    });
+
     nvim.quit();
 
     proc.on('close', () => {
```

Comment:
Based on local runs, changing this to 'exit' seems more reliable. It's strange that 'close' is sometimes not emitted, but that was never the purpose of this test...
```suggestion
    proc.on('exit', () => {
```

Reply:
Make sense, we only want make sure disconnect is called.